Update readme

cdrage opened the pull request:
* Removes mentions of install for fetch
* New year, new copyright update
* Cleans up some of descriptions
* Remove grammatical errors

cdrage commented on README.md:
````diff
@@ -25,47 +27,40 @@ First of all, clone the github repository: `git clone https://github.com/project
-```
-cd atomicapp
-export PYTHONPATH=`pwd`:$PYTHONPATH
-alias atomicapp="python `pwd`/atomicapp/cli/main.py"
````
i don't think this should be included ^^. let python devs do their own way of development

dustymabe replied:
I don't mind having it. Sometimes I use this and I always come back here as a reference.

cdrage commented:
pingarino @goern @vpavlin since you two did most of the initial README work according to git logs :)

cdrage commented:
Ping @dustymabe @rtnpro for review as this part is old stuff ^^ we don't use `install` anymore.

dustymabe commented on README.md:
```diff
@@ -25,47 +27,40 @@ First of all, clone the github repository: `git clone https://github.com/project
-Just a call to Docker to package up the application and tag the resulting image.
+Use 'docker build' to package up the application and tags the resulting image.
 
 ### Install and Run
```
Install --> Fetch ??

dustymabe commented on README.md:
````diff
@@ -25,47 +27,40 @@ First of all, clone the github repository: `git clone https://github.com/project
 ### Install and Run
 ```
-atomicapp [--dry-run] [-a answers.conf] install|run [--recursive] [--update] [--destination DST_PATH] APP|PATH
+atomicapp [--dry-run] [-a answers.conf] fetch|run|stop [--recursive] [--update] [--provider docker] [--destination DST_PATH] APP|PATH
````
There is also genanswers. I think maybe we should not put all of the options here because it gets too long.

dustymabe commented on README.md:
```diff
@@ -25,47 +27,40 @@ First of all, clone the github repository: `git clone https://github.com/project
 existing path, it looks for `Nulecule` file there instead of pulling anything.
 
 * `--recursive yes|no` Pull whole dependency tree
 * `--update` Overwrite any existing files
```
are these two options really useful? I wonder if we shouldn't get rid of them

cdrage replied:
yeah, you're right, I don't think I've actually used either of them once.

dustymabe commented:
Other than comments it looks good to me.

cdrage commented:
@dustymabe updated with your comments.

dustymabe commented on README.md:
````diff
@@ -36,36 +38,35 @@ export PYTHONPATH=`pwd`:$PYTHONPATH
 ```
 
-### Build
+### Build an Atomic App
````
Can we be more clear about what this is doing? "Build an Atomic App" sounds like I am doing an application build (i.e. helloapache) vs packaging up the atomicapp software. Maybe this would be better: `Build an Atomic App Container Image - for Container Execution`. And then mention running it via Atomic CLI? 

And maybe we even want to change the name of the previous section to `Install Atomic App - for Local Execution`

dustymabe commented on README.md:
````diff
@@ -36,36 +38,35 @@ export PYTHONPATH=`pwd`:$PYTHONPATH
 ```
 
-Just a call to Docker to package up the application and tag the resulting image.
+Use 'docker build' to package up the application and tags the resulting image.
````
s/tags/tag/

dustymabe commented:
@cdrage - can we get this updated?

cdrage commented:
Hey @dustymabe 
So I updated it with the new title as well as the spelling error.

In regards to `atomic cli` execution. Should we add a separate section for alternatively using the atomic CLI? 

Otherwise, rebased again ^^

dustymabe commented:
`Build an Atomic App for containerized execution` still sounds wrong. 

What we are doing is containerizing the Atomic App software so it can then be run in a container but it isn't actually an "Atomic App". `Building for containerized execution` would be a better title. We should probably update `Install this project` to `Install Atomic App Software Locally` or something along those lines. 

I say just digest these comments and update it based on your best judgement and then MERGE!!

cdrage commented:
@dustymabe 
Updated based on your comments!

So I'm going to push this PR for now and I'll be adding a separate section for "atomic cli" execution this week. Work on doc updating for our upcoming release :)

surajssd commented:
I think work on #543 can continue :)
